File: BackEnd/postman/Estadisticas.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Estadisticas'
data={
  'accion':'medicos',
}
print postear(data,url)
print

url='Estadisticas'
data={
  'accion':'centros',
}
print postear(data,url)
print

url='Estadisticas'
data={
  'accion':'pacienteMax',
}
print postear(data,url)
print

File: BackEnd/ajax/Estadisticas.js
$.ajax({
  url:CONST_SITIO_URL+'/services/Estadisticas.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'medicos',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Estadisticas.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'centros',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Estadisticas.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'pacienteMax',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


